index.html

```diff
@@ -41,4 +41,140 @@ <h2 class="h2_title">I am a Software Developer.</h2>
 <li class="social_media_icons"> <a href="#"><img src="image folder/twitter.svg" alt=""></a></li>
 <li class="social_media_icons"> <a href="#"><img src="image folder/medium.svg" alt=""></a></li>
   </ul>
-</section>
\ No newline at end of file
+</section>
+<section class="recent_works_section">
+  <div class="recent_work_div flex">
+    <h2 class="recent_work_header">My Recent Works</h2>
+    <img class="image_line" src="image folder/recent_work_line.png" alt="line">
+   
+  </div>
+ 
+  <div class="recent_work_content">
+      <img class="img_placeholder" src="image folder/ImgPlaceholder.png" alt="frame">
+    <h3 class="multi_story">Multi-Post Stories</h3>
+    <p class="recent_work_text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+    <ul class="recent_work_language flex">
+      <li class="work_language flex">css</li>
+      <li class="work_language flex">html</li>
+      <li class="work_language flex">Bootstrap</li>
+      <li class="work_language flex">Ruby</li>
+    </ul>
+    <button class="see_prjt_buttn flex">See Project</button>
```

Comment:
- [ ] It looks like you have not removed the border from all the `See Projects` buttons. Please kindly fix the button design to make it similar to the Figma template. Please try to stick with your selected design template for better UI and consistency. 👍🏽 😃 

| Figma Design | Your Design |
|------------------- | ------------------- |
| ![Figma_1](https://user-images.githubusercontent.com/24832846/205331757-1bc560eb-0502-4675-8701-e1f55c728890.png) | ![Your](https://user-images.githubusercontent.com/24832846/205331795-c8ae83dd-d135-4857-8bcb-9a690105a280.png) |